lib/spring/application.rb

```diff
@@ -147,7 +147,7 @@ def serve(client)
       log "got client"
       manager.puts
 
-      stdout, stderr, stdin = streams = 3.times.map { client.recv_io }
+      _stdout, stderr, _stdin = streams = 3.times.map { client.recv_io }
```

Comment:
Seems we aren't using these local variables at all. Can we just remove them?

Reply:
We are using the `stderr` in the `rescue` block